### DISCUSSION TITLE
Strictness level on command line local AntiSMASH 7.1.0

### DISCUSSION THREAD

**Discussion by ciroariel** [antismash/antismash#706] [Q&A] [answered]
I'd like to know how can I run AntiSMASH 7.1.0 on my ubuntu command line using the strictiness level --strict (instead of default --relaxed mode). I have already searched within the --help options and there is nothing about it. I read an alternative on Chat GPT that it could be done by changing the configuration file in AntiSMASH folder, but I really dont wanna make permanently changes in software configs.
Do someone have any ideas?

**Comment by SJShaw** [comment, marked as answer]
This is the part you're looking for:
```
$ antismash --help-showall

...
HMM detection options:

  --hmmdetection-strictness {strict,relaxed,loose}
                        Defines which level of strictness to use for HMM-based cluster detection, (default: relaxed).
```

**Reply by ciroariel** [reply]
Perfect! 
When I saw this in Options, I was wondering if it isn't a strictness level only for this specific analysis (HMM), not at a general manner. But, looking carefully, that's exactly what I need, isn't?!
Thank's for the answer.
